GenerateAndSendStorageReport.py

```diff
@@ -13,9 +13,9 @@
 currentDate = datetime.datetime.now().isoformat()[:10]  # YYYY-MM-DD
 currentTime = datetime.datetime.now().isoformat()[:19]
 
-def createAndSendReport(email, apiKey, recipientIds, tableId):
+def createAndSendReport(email, token, recipientIds, tableId):
```

Comment:
Nit: you can technically remove email all together as well, but its fine to keep it.

Reply:
i'll let the original author, @nickgros, decide there.

Reply:
Agree with @thomasyu888